### DISCUSSION TITLE
Failed to fetch dynamically imported module

### DISCUSSION THREAD

**Discussion by cyperdark** [nuxt/nuxt#16267] [Questions]
Hello, i checked my site in google robot and it seems he cant proceed the page because of ``import`` thing in ``.js`` files
![image](https://user-images.githubusercontent.com/21735205/204154245-0dce4b62-5348-4f46-b5b7-98ce929b4ab5.png)

so how i can change it to commonjs?

Google bot result: [click here](https://search.google.com/test/rich-results/result?id=nWd0FTL8esf_8kJ9qVKGew)

![image](https://user-images.githubusercontent.com/21735205/204154250-5558f571-7c8e-4334-9523-da751bbab3c5.png)

**Comment by TheAlexLichter** [comment]
Could be related to https://github.com/nuxt/framework/issues/6648
